docs/topics/db/models.txt

```diff
@@ -907,18 +907,13 @@ example::
         name = models.CharField(max_length=100)
         slug = models.TextField()
 
-        def save(
-            self, force_insert=False, force_update=False, using=None, update_fields=None
-        ):
+        def save(self, *args, **kwargs):
             self.slug = slugify(self.name)
-            if update_fields is not None and "name" in update_fields:
-                update_fields = {"slug"}.union(update_fields)
-            super().save(
-                force_insert=force_insert,
-                force_update=force_update,
-                using=using,
-                update_fields=update_fields,
-            )
+            if kwargs.get("update_fields") is not None and "name" in kwargs.get(
```

Comment:
It might be nicer to use the `:=` operator here to avoid drilling into kwargs twice, along the lines of [this](https://forum.djangoproject.com/t/update-or-create-behavior/25944/4).